packages/core/src/components/modal/modal.tsx

```diff
@@ -143,9 +143,9 @@ export class AtomModal {
               ></atom-icon>
             </atom-button>
           </header>
-          {!!this.progress && (
+          {this.progress >= 0 ? (
```

Comment:
isso é daquele bug que me mostrou?

Reply:
isso, era o cenário em que a barra não renderizava no valor zero

Reply:
@jowjow22 we need to add some tests about it